webxrlayers-1.bs

```diff
@@ -921,7 +928,10 @@ To <dfn>allocate color textures</dfn> using an {{XRCompositionLayer}} |layer|, a
           <dd> Initialize |array| with 1 [=new=] instance of {{WebGLTexture}} in the [=relevant realm=] of |context| created as a {{TEXTURE_2D_ARRAY}} texture with |numViews| internal textures using |context| and |init|'s {{XRLayerInit/alpha}}, {{XRLayerInit/viewPixelWidth}} and {{XRLayerInit/viewPixelHeight}} values.
           <dd> Return |array| and abort these steps.
         <dt> Otherwise
-          <dd> Initialize |array| with |numViews| [=new=] instances of {{WebGLTexture}} in the [=relevant realm=] of |context| created as a {{TEXTURE_2D}} texture with |context| and |init|'s {{XRProjectionLayerInit/alpha}}, {{XRLayerInit/viewPixelWidth}} and {{XRLayerInit/viewPixelHeight}} values.
+          <dd> For each |view| in session's {{XRView|XRViews}}:
```

Comment:
ditto, you want https://immersive-web.github.io/webxr/#xrsession-viewer-reference-space-list-of-views

Reply:
Do you know how I can add a link to that from another spec?

Reply:
Link to `list of views`, and add `type: dfn; text: list of views; for: XRSession/viewer reference space` to the link defaults section